problems/13.Merge Sorted Array.gs

```diff
@@ -0,0 +1,35 @@
+/**
+ Do not return anything, modify nums1 in-place instead.
+ */
+function merge(nums1: number[], m: number, nums2: number[], n: number): void {
+
+    for (let i = 0; i < n; i++) {
+        nums1[m + i] = nums2[i]
+    }
+
```

Comment:
Please fix white space in between things. Here you have for some reason 3 new lines and then again after "sortiraa" 2 more. Why exactly - it looks really bad please fix it.